public/components/custom_panels/helpers/add_visualization_helper.ts

```diff
@@ -58,7 +58,7 @@ export const addVisualizationPanel = (
   // client: ILegacyScopedClusterClient,
   // panelId: string,
   savedVisualizationId: string,
-  oldVisualizationId?: string,
+  oldVisualizationId: string | undefined,
```

Comment:
Maybe... I think what it is doing is makes it optional, which it cannot be since a required parameter comes after. This gets rid of it but maybe the more right thing is to put it at the end...